.github/ISSUE_TEMPLATE

```diff
@@ -0,0 +1,20 @@
+# Issue Description
+  [Type of Issue]
+
+## Problem
+  [Describe what the problem we are trying yo solve.]
```

Comment:
I think you have a typo here.

Yo should be `to`

Maybe should say

`Describe the problem we are trying to solve`

Reply:
Thank you for catching this typo.